AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/firebase/FirebaseTestLabApi.kt

```diff
@@ -65,26 +66,34 @@ internal interface FirebaseTestLabApi {
     ): GetApkDetailsResponse
 
     companion object {
+        private const val MAX_CONCURRENT_REQUESTS = 64
```

Comment:
I dont think we want to hard-code it for all users of androidx-ci-action (like androidx/androidx github). We should make it configurable by the user of this library.

Reply:
It could either take a config value or a dispatcher with a default value.

Reply:
Done